### PR TITLE
Add preliminary devel env info to README.md.

### PR DESCRIPTION
**Pull request by fisx**

### CODE REVIEW

**Comment by fisx** [comment]
note this approach doesn't support simultaneously hacking thentos and aula.  we're working on that.

**Comment by fisx** [comment]
> [...] support simultaneously hacking thentos and aula. we're working on that.

worked!  (:

File: README.md [hunk 1]
```diff
@@ -4,6 +4,52 @@ status: just brainstorming
...
+
+if you want to hack both thentos and aula, `/root/thentos` is exposed
+as a docker volume similarly to `/root/aula`.  Just duplicate the `-v`
+option to `docker run`.
+
+
+## Getting started (with sensei)
+
+https://github.com/hspec/sensei is a tool for rapid re-compilation and
+testing.  It works with .  If you want to use it, follow these steps:
```
Comment (np) [inline README.md]: missing word?

Comment (Mikolaj) [inline README.md]: @fisx: did you fix it? you said "11:55 < fisx_> npouillard: not sure what that sentence was for.  i removed it.  and added one more paragraph." but perhaps you didn't push?

**Pull request merged**